net-misc/dropbox/dropbox-45.3.88.ebuild

```diff
@@ -7,7 +7,7 @@ PYTHON_COMPAT=( python2_7 python3_5 python3_6 )
 inherit eutils gnome2-utils pax-utils systemd python-single-r1
 
 DESCRIPTION="Dropbox daemon (pretends to be GUI-less)"
-HOMEPAGE="http://dropbox.com/"
+HOMEPAGE="https://www.dropbox.com/"
```

Comment:
Why are you adding the `www.`?

Reply:
Because http(s)://dropbox.com eventually redirects to https://www.dropbox.com anyway. So in order to save a redirect I added the www part to the URL.